dev-docs/show-outstream-video-ads.md

```diff
@@ -67,13 +65,16 @@ To display an outstream video, two things are needed:
 
 Prebid.js will select the `renderer` used to display the outstream video in the following way:
 
-1. If a `renderer` is associated with the Prebid adUnit, it will be used to display any outstream demand associated with that adUnit.  Below, we will provide an example of an adUnit with an associated `renderer`. If that renderer is specified as backup only, it will only be used when no other renderer is found. 
-2. If no `renderer` is specified on the Prebid adUnit, Prebid will invoke the renderer associated with the winning (or selected) demand partner video bid.  Choosing a backup only renderer allows publishers to access demand with or without an attached renderer.
+1. If a `renderer` is associated with the Prebid adUnit's video mediaType, it will be used to display any outstream demand associated with that adUnit with a mediaType of "video".
+2. If a `renderer` is associated with the Prebid adUnit, it will be used to display any outstream demand associated with that adUnit.  Below, we will provide an example of an adUnit with an associated `renderer`.
```

Comment:
I would again remove the word outstream here, and perhaps indicate that the url would be invoked regardless of mediaType. Also, I think it makes sense to say this is legacy and #1 is now the preferred way.